pipeline-runner/R/qc-helpers.R

```diff
@@ -1,3 +1,42 @@
+#' Title
+#'
+#' @param scdata
+#'
+#' @return
+#' @export
+#'
+#' @examples
+generate_first_step_ids <- function(scdata) {
+  cells_id <- list()
+  for (sample_id in unique(scdata$samples)) {
+    cells_id[[sample_id]] <- scdata$cells_id[scdata$samples == sample_id]
+  }
+  return(cells_id)
+}
+
+#
+# Returns the positions to keep based on scdata and number of cells to keep
+#
+get_positions_to_keep <- function(scdata, num_cells_to_downsample) {
+  # Downsample plotData
+  num_cells_to_downsample <- downsample_plotdata(ncol(scdata), num_cells_to_downsample)
```

Comment:
do we use `downsample_plotdata` anywhere else still? If not, might make sense to just pull it into this function as its pretty small and poorly named